mitodl/ocw-studio
# Include uploaded filepath in backend (github) metadata

Blocked by / based on #329 

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #333 

#### What's this PR do?
- Modifies the content_sync serializer to include a `WebsiteContent.file` url in the metadata if it exists.
- Sets uploaded file ACL permissions to `public-read`

#### How should this be manually tested?
- Create a new `WebsiteContent` object with a file upload field and save
- Check that the uploaded file url is saved to github
- Check that you can view the S3 URL without getting an "Access Denied" message
- Edit the content above, and change just the file to another file, then save.
- Check that the uploaded file url is also updated in github
- Check that you can view the S3 URL without getting an "Access Denied" message
